clusters/app.ci/cert-manager/cert-issuer-staging_clusterissuer.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: cert-issuer-staging
+spec:
+  acme:
+    email: openshift-ci-robot@redhat.com
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
```

Comment:
Maybe change it to prod before we merge?

Reply:
We have 2 clusterissuers: prod. and staging.